expressions/aggregation/AggregationHandleAvg.hpp

```diff
@@ -156,11 +217,12 @@ class AggregationHandleAvg : public AggregationConcreteHandle {
    */
   void aggregateOnDistinctifyHashTableForGroupBy(
       const AggregationStateHashTableBase &distinctify_hash_table,
-      AggregationStateHashTableBase *aggregation_hash_table) const override;
+      AggregationStateHashTableBase *aggregation_hash_table,
+      int index) const override;
 
-  void mergeGroupByHashTables(
-      const AggregationStateHashTableBase &source_hash_table,
-      AggregationStateHashTableBase *destination_hash_table) const override;
+  size_t getPayloadSize() const override {
```

Comment:
It is considered a good practice to use the ``std`` prefix in the header file. Therefore, can you please change the ``size_t`` to ``std::size_t``?